Mathlib/RingTheory/Polynomial/Eisenstein/Criterion.lean

```diff
@@ -165,9 +165,9 @@ theorem generalizedEisenstein {q f : R[X]} {p : ℕ}
       exact (dvd_pow_self q hn).mul_left _
     · exact ((dvd_pow_self q hn).mul_left _).mul_left _
 
-/-- If `f` is a non constant polynomial with coefficients in `R`, and `P` is a prime ideal in `R`,
+/-- If `f` is a nonconstant polynomial with coefficients in `R`, and `P` is a prime ideal in `R`,
```

Comment:
dito regarding hyphens

Reply:
See the general comment below.

Reply:
https://www.merriam-webster.com/dictionary/nonconstant

A listing in MW means I'll be very reluctant to add a hyphen here.